framework/console/controllers/BaseMigrateController.php

```diff
@@ -465,8 +505,19 @@ public function actionNew($limit = 10)
      * yii migrate/create create_user_table
      * ```
      *
+     * In order to generate migration with namespace, you should add it before migration self name.
```

Comment:
In order to generate namespaced migration you should specify namespace before migration's name.